src/promptflow/promptflow/_sdk/operations/_test_submitter.py

```diff
@@ -190,11 +194,14 @@ def node_test(
         from promptflow.executor import FlowExecutor
 
         connections = SubmitterHelper.resolve_connections(flow=self.flow, client=self._client)
+        credential_list = ConnectionManager(connections).get_secret_list()
+
         # resolve environment variables
         SubmitterHelper.resolve_environment_variables(environment_variables=environment_variables, client=self._client)
         SubmitterHelper.init_env(environment_variables=environment_variables)
 
-        with LoggerOperations(file_path=self.flow.code / PROMPT_FLOW_DIR_NAME / f"{node_name}.node.log", stream=stream):
+        with LoggerOperations(file_path=self.flow.code / PROMPT_FLOW_DIR_NAME / f"{node_name}.node.log",
+                              stream=stream, credential_list=credential_list):
```

Comment:
Add a test case for this?

Reply:
Added sdk cli test to ensure the secret message in the flow and node test will be redacted.